apps/site/src/app/(home)/sections/Involved/Involved.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+
+import anteater_left from "@/assets/images/involved_anteater_left.svg";
+import anteater_right from "@/assets/images/involved_anteater_right.svg";
+
+import styles from "./Involved.module.scss";
+
+import { motion } from "framer-motion";
+
+const Mentor = () => {
+	const sectionHeader = <h2 className={styles.headerText}>GET INVOLVED</h2>;
+	const leftBubbleText = <p>Want to develop your first project?</p>;
+	const rightBubbleText = (
+		<p>Otherwise, if you have some experience under your belt,</p>
+	);
+	const applyLink = (
+		<Button href="#" type="button" className={styles.applyButton}>
+			Apply as a Hacker
+		</Button>
+	);
+
+	const mentorLink = (
+		<Button href="#" type="button" className={styles.applyButton}>
+			Apply as a Mentor
+		</Button>
+	);
+
+	const hoverAnimation = {
+		scale: 1.05,
+		rotate: [0, 10, -10, 0],
+	};
+
+	const hoverTransition = {
+		duration: 0.5,
+		ease: "easeInOut",
+	};
```

Comment:
Question: is there a particular reason you decided to implement these visual effects with Framer? The motion here should be relatively simple to implement with native [CSS animations](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animations/Using_CSS_animations).

Reply:
we were already using framer somewhere else, so it was convenient, but i've gone ahead and converted them to native CSS animations